scvelo/plotting/utils.py

```diff
@@ -1465,15 +1476,14 @@ def log_fmt(x, pos):
     pdf = [pdf] if isinstance(pdf, str) else pdf
     if pdf is not None:
         fits = []
-        for i, pd in enumerate(pdf):
-            from scipy import stats
-
+        for i, pdf_name in enumerate(pdf):
             xt = ax.get_xticks()
             xmin, xmax = min(xt), max(xt)
             lnspc = np.linspace(xmin, xmax, len(bins))
 
-            if "(" in pd:  # used passed parameters
-                args, pd = eval(pd[pd.rfind("(") :]), pd[: pd.rfind("(")]
+            if "(" in pdf_name:  # used passed parameters
+                start = pd.rfind("(")
```

Comment:
Also change to `pdf_name.rfind` (this is not a pandas function, but applied to a string).

Reply:
Yes, thanks. Good catch!